src/components/NavBar.js

```diff
@@ -0,0 +1,30 @@
+import { Navbar, Nav, Container } from "react-bootstrap";
+import React from "react";
+import navIcon1 from '../assets/img/block-linkedin.png';
+import navIcon2 from '../assets/img/block-github.png';
+import navIcon3 from '../assets/img/block-fb.png';
+import logo from '../assets/img/logo.png';
+
+export const NavBar = () => {
+  return (
+    <Navbar expand="lg">
+      <Container>
+        <Navbar.Brand href="#home">
+          <img src={logo} alt="Logo" className="logo-img"/>
+        </Navbar.Brand>
+        <Navbar.Collapse id="basic-navbar-nav">
+          <Nav className="ms-auto">
+            <Nav.Link href="#home" className='navbar-link-home'></Nav.Link>
+            <Nav.Link href="#skills" className='navbar-link-skills'></Nav.Link>
+            <Nav.Link href="#projects" className='navbar-link-projects'></Nav.Link>
+          </Nav>          
+          <div className="social-icon">
+            <a href="https://www.linkedin.com/in/bsmonton/"><img src={navIcon1} alt="" /></a>
```

Comment:
The img should have a alt value.
alt is shown in place of image if it fails to load, 
so that the user will know what that image is for
for example we can put "Linkedin" or "To Linkedin" on the alt 
since its purpose is to direct it to Linkedin
Also for better user experience, 
clicking on this button should open the link on another tab